### ISSUE TITLE
Microformats h-feed support

### ISSUE THREAD

**Issue by gRegorLove** [fguillot/picoFeed#264]
Hi all,

I was interested in running miniflux on my site and would like to add support for parsing [microformats h-feed](http://microformats.org/wiki/h-feed). I thought I would post here to see if there's any interest in a PR to add that functionality in picoFeed.

**Comment by BernhardPosselt** [comment]
If I understand this correctly, you basically use css classes and html structures to mark up feeds. Feels very experimental, how mature and wide spread is this?

**Comment by gRegorLove** [comment]
Microformats are coming up on 10-years-old. They're baked into the default WordPress themes, and many other themes, so a lot of sites have microformats even if the author is not aware of it. A lot of us in the  [indiewebcamp community](http://indiewebcamp.com/irc-people) use microformats for feeds since it's more DRY.

Microformats2 (mf2) is the latest interoperable version and there are parsers in most programming languages. [php-mf2](https://github.com/indieweb/php-mf2) is the parser I'd be implementing. http://pin13.net running php-mf2, and here it is parsing an h-feed on my site: http://pin13.net/mf2/?url=http%3A%2F%2Fgregorlove.com%2Fnotes%2F 

Let me know what you think!

**Comment by BernhardPosselt** [comment]
Since they are less used than Atom and RSS I think adding support would be great but we should configure the parser to discover it after Atom and RSS.

My guess is that especially in the case of Wordpress unexperienced web developers can actually break the convention by deviating a bit from the standard theme. If we parse this as with a higher importance than Atom and RSS we are likely to receive a worse overall experience.

Basically: Atom > RSS > microformats

@fguillot @mkresin your thoughts :)?

**Comment by gRegorLove** [comment]
I started some very basic tinkering with adding mf2 support, adding an Xpath to detect "h-entry" after the Atom/RSS xpaths. It seems to be working.

**Comment by fguillot** [comment]
Well, if you feel confortable to add a new parser (with tests) for micro-formats h-feed, why not.
